config/examples/sim-elf-scattered.config

```diff
@@ -0,0 +1,28 @@
+ARCH=sim
```

Comment:
Could there be build tests for these new example configs?

Reply:
good idea. I'm adding a test now. Thanks @dgarske